recipes/ambertools/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "AmberTools" %}
+{% set version = "19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/simonbray/ambertools/raw/9e3a9289ca97a2ba0f58cebdfb102c88e8912d9f/AmberTools19.tar.bz2
```

Comment:
please add a description why this URL is needed, in contrast to the original URL